graal-js/src/com.oracle.truffle.js/src/com/oracle/truffle/js/nodes/control/GeneratorWrapperNode.java

```diff
@@ -78,6 +78,9 @@ public boolean hasTag(Class<? extends Tag> tag) {
         if (tag == JSTags.ControlFlowBranchTag.class) {
             return true;
         }
+        if (tag == JSTags.InputNodeTag.class) {
+            return true;
+        }
```

Comment:
I don't think that generator wrappers are _always_ input tags -- we might have cases where they are not (maybe for generators, I don't know) -- so perhaps it's better to return true only if `this instanceof XXX` where `XXX` is what you need here -- I'm not sure this is possible, but please check

Reply:
ok, done.